{"version":1,"title":"KASAN: global-out-of-bounds Read in pid_task (3)","id":"841c99985a4f48b3bcf208cbcd64ec66e607d2bc","discussions":["https://groups.google.com/d/msgid/syzkaller-upstream-moderation/68cb108c.050a0220.3c6139.0fa8.GAE@google.com"],"crashes":[{"title":"KASAN: global-out-of-bounds Read in pid_task","kernel-config":"/text?tag=KernelConfig\u0026x=12a1d1f3a8199632","kernel-source-git":"https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/log/?id=590b221ed4256fd6c34d3dea77aa5bd6e741bbc1","kernel-source-commit":"590b221ed4256fd6c34d3dea77aa5bd6e741bbc1","syzkaller-git":"https://github.com/google/syzkaller/commits/e2beed91937c0ace342f19a2e9afb67adb3a828a","syzkaller-commit":"e2beed91937c0ace342f19a2e9afb67adb3a828a","crash-report-link":"/text?tag=CrashReport\u0026x=14a20b62580000"}]}